pipeline/beam_tables.py

```diff
@@ -182,23 +205,25 @@ def _get_partition_params() -> Dict[str, Any]:
   return partition_params
 
 
-def get_job_name(table_name: str, incremental_load: bool) -> str:
+def get_job_name(destination: str, incremental_load: bool) -> str:
```

Comment:
same here. I would break this into two methods, maybe with a shared helper.

Reply:
done